third_party/OpenAPI/gotocompany/guardian/v1beta1/guardian.swagger.json

```diff
@@ -3656,6 +3681,9 @@
         "permissions": {
           "type": "array",
           "items": {}
+        },
+        "type": {
```

Comment:
Do we still need this?

Reply:
wait, I did not add this but it came up on this PR
this is the proton for policy changes https://github.com/goto/proton/pull/139
and looks like this field was introduced on this PR https://github.com/goto/proton/pull/138/files
tried to rebase but it says it's already updated hmm

Reply:
Oh I guess this one got added on proton by gtf but currrently not implemented on our guardian. That's why on my addition of new param, it also got added there. This should be ok for now

Reply:
Let’s make sure it doesn’t break anything.